sycl/source/detail/platform_impl.hpp

```diff
@@ -74,10 +74,10 @@ class platform_impl {
   bool is_host() const { return MHostPlatform; };
 
   bool is_cuda() const {
-    const string_class CUDA_PLATFORM_STRING = "NVIDIA CUDA";
+    const string_class CUDA_PLATFORM_STRING = "NVIDIA CUDA BACKEND";
     const string_class PlatformName =
-        get_platform_info<string_class, info::platform::name>::get(MPlatform,
-                                                                   getPlugin());
+        get_platform_info<string_class, info::platform::version>::get(
```

Comment:
My understanding is that the reported version contains the string `CUDA 10.2....` do you mean to change from `info::platform::name` to `info::platform::version` here?

Reply:
That was an accident, fixed now. Thanks